CMakeLists.txt

```diff
@@ -259,14 +259,12 @@ macro (config_hook)
 
   if(hip_FOUND)
     get_filename_component(CXX_COMPILER ${CMAKE_CXX_COMPILER} NAME)
-    if(${CXX_COMPILER} STREQUAL "hipcc")
-      message(FATAL_ERROR " hipcc does not work, use g++ or clang++")
-    endif()
     if(hip_VERSION VERSION_LESS "5.3")
-      if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
-        message(WARNING " Cannot use g++ for rocsparse with rocm < 5.3\n Disabling rocsparseSolver")
-        unset(HAVE_ROCSPARSE)
-      endif()
+	    if(NOT CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
```

Comment:
`.... AND ROCALUTION_FOUND)`?

Reply:
I added that to the check, and moved the rocalution adding outside of the if statement.

Reply:
Can you start jenkins again?